pkg/apis/flyteworkflow/v1alpha1/admin.go

```diff
@@ -0,0 +1,15 @@
+package v1alpha1
+
+import "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+
+// This contains an OutputLocationPrefix. When running against AWS, this should be something of the form
+// s3://my-bucket, or s3://my-bucket/  A sharding string will automatically be appended to this prefix before
+// handing off to plugins/tasks. Sharding behavior may change in the future.
+// Background available at https://github.com/lyft/flyte/issues/211
+type RawOutputDataConfig struct {
+	*admin.RawOutputDataConfig
+}
+
+func (in *RawOutputDataConfig) DeepCopyInto(out *RawOutputDataConfig) {
```

Comment:
did you write it or is this auto-generated?

Reply:
haytham told me to write this.  the reason is so that in the future, we can add things like enums if we want without breaking prior versions.